src/settings/FinePolicy/components/EditSections/FinesSection/FinesSection.test.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {
+  render,
+} from '@testing-library/react';
+
+import '../../../../../../test/jest/__mock__';
+
+import { Accordion } from '@folio/stripes/components';
+import OverdueFinesSection from '../RangeSection/OverdueFinesSection';
+import OverdueFinesSectionColumn from '../RangeSection/OverdueFinesSectionColumn';
+import FinesSection from './FinesSection';
+
+const mockTestGeneratedOptions = [
+  <option
+    value="testValue1"
+    key="key1"
+  >
+    testValue1
+  </option>,
+  <option
+    value="testValue2"
+    key="key2"
+  >
+    testValue2
+  </option>,
+];
+
+jest.mock('../RangeSection/OverdueFinesSection', () => jest.fn(() => null));
+jest.mock('../RangeSection/OverdueFinesSectionColumn', () => jest.fn(() => null));
+jest.mock('../../../../utils/options-generator', () => jest.fn(() => mockTestGeneratedOptions));
+
+describe('FinesSection', () => {
+  const labelIds = {
+    accordion: 'ui-circulation.settings.finePolicy.overdueFine',
+    overdueFineSection: 'ui-circulation.settings.finePolicy.overdueFine',
+    countClosedDHMColumn: 'ui-circulation.settings.finePolicy.countClosedDHM',
+    maximumOverdueFineColumn: 'ui-circulation.settings.finePolicy.maximumOverdueFine',
+    forgiveOverdueFineColumn: 'ui-circulation.settings.finePolicy.forgiveOverdueFine',
+    overdueRecallFineSection: 'ui-circulation.settings.finePolicy.overdueRecallFine',
+    ignoreGracePeriodsRecallsColumn: 'ui-circulation.settings.finePolicy.ignoreGracePeriodsRecalls',
+    maximumRecallOverdueFineColumn: 'ui-circulation.settings.finePolicy.maximumRecallOverdueFine',
+  };
+
+  describe('when "fineSectionOpen" prop is true', () => {
+    beforeEach(() => {
+      render(<FinesSection fineSectionOpen />);
+    });
+
+    it('should render "Accordion" component with passed props', () => {
+      expect(Accordion).toHaveBeenCalledWith(expect.objectContaining({
+        id: 'editFineSection',
+        open: true,
+        label: labelIds.accordion,
+      }), {});
+    });
+
+    it('should render overdue fine section', () => {
```

Comment:
i think we should write the section title in quotes. "overdue fine" for example. it looks more comfortable for reading, isn't it?

Reply:
yeah, good point. done.